lib/fog/backblaze/token_cache.rb

```diff
@@ -22,11 +22,6 @@
 #   Fog::Storage.new(provider: 'backblaze', ..., token_cache: RedisTokenCache.new)
 #
 
-module Fog
-  module Backblaze
-  end
-end
-
```

Comment:
This isn't need anymore now that we are using `autoload` and making sure that everything is in place and loaded in the right order.